### PR TITLE
uint: bump version to 0.7

### PR DESCRIPTION
**Pull request by ordian**
to incorporate #126

### CODE REVIEW

File: ethereum-types/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ethereum-types"
-version = "0.5.0"
+version = "0.5.1"
```
Comment (ascjones) [inline ethereum-types/Cargo.toml]: So the policy for dependent crates is to bump the the patch when updating a dependency?

Comment (dvdplm) [reply ethereum-types/Cargo.toml]: I don't think we have a hard policy on that but I think it's a good practise if applied sensibly? Do you see any downsides?

Comment (ascjones) [reply ethereum-types/Cargo.toml]: Seems fine to me - was just curious what our convention was

Comment (ordian) [reply ethereum-types/Cargo.toml]: I think that depends on whether the change is breaking, I don't think it is of ethereum-types and primitive-types, but please correct me if I'm wrong.

Comment (ordian) [reply ethereum-types/Cargo.toml]: Or if you mean in general, I don't think we should do this in every case, but I'd like to publish new version of primitive-types as well.

Comment (ascjones) [reply ethereum-types/Cargo.toml]: Yes of course my original comment is poorly phrased. Just when I've updated a dependency like this in the past (non breaking minor version) I was unsure what was the correct bump for the parent crate.

**Pull request merged**